app/util/HelperFunctions.scala

```diff
@@ -39,15 +38,33 @@ object HelperFunctions {
     Either.cond(body.isDefined, body.get, BodyRequiredForUpdateError)
   }
 
-  def parseAtomJson(atomJson: String): Either[AtomAPIError, Atom] = {
-    Logger.info(s"Parsing atom json: ${atomJson}")
+  def parseStringToAtom(atomString: String): Either[AtomAPIError, Atom] = {
```

Comment:
It doesn't need to be in this PR (or at all) but at some point we could add some tests around the helper functions taking strings as params as that would be good documentation of what a valid string looks like